src/index.pug

```diff
@@ -37,33 +37,38 @@ html(lang="en")
 			table
 				thead
 					tr
+						th(scope="col") Region
 						th(scope="col") Domain
 						th(scope="col") Health
 						th(scope="col") Action
 				tbody#instances-tbody
 					tr.loading
-						td(colspan="3" data-loading-message="Loading instances...").loading-td Trying to run scripts...
+						td(colspan="4" data-loading-message="Loading instances...").loading-td Trying to run scripts...
 
 		noscript
 			.instances-list
 				h2 Available instances
 				ul.list
 					-
 						const instances = [
-							"invidious.snopyta.org",
-							"yewtu.be",
-							"invidious.kavin.rocks",
-							"invidious-us.kavin.rocks",
-							"invidious-jp.kavin.rocks",
-							"vid.puffyan.us",
-							"invidious.namazso.eu",
-							"inv.riverside.rocks",
-							"vid.mint.lgbt",
-							"invidious.osi.kr",
-							"invidio.xamh.de",
-							"yt.artemislena.eu",
+							{url: "yewtu.be", flag: "🇳🇱"},
+							{url: "vid.puffyan.us", flag: "🇺🇸"},
+							{url: "invidious.flokinet.to", flag: "🇷🇴"},
+							{url: "invidious.projectsegfau.lt", flag: "🇫🇷"},
+							{url: "inv.bp.projectsegfau.lt", flag: "🇱🇺"},
+							{url: "inv.in.projectsegfau.lt", flag: "🇮🇳"},
+							{url: "invidious.tiekoetter.com", flag: "🇩🇪"},
+							{url: "invidious.slipfox.xyz", flag: "🇺🇸"},
+							{url: "invidious.privacydev.net", flag: "🇫🇷"},
+							{url: "vid.priv.au", flag: "🇸🇬"},
+							{url: "iv.ggtyler.dev", flag: "🇨🇦"},
+							{url: "invidious.0011.lt", flag: "🇱🇹"},
+							{url: "inv.zzls.xyz", flag: "🇨🇱"},
+							{url: "invidious.protokolla.fi", flag: "🇫🇮"},
 						]
 					for instance in instances
-						li: a(href=`https://${instance}`)= instance
+						li
+							= `instance.flag `
```

Comment:
I just tested this locally. Please change this to ``= `${instance.flag} ` `` (adding javascript string interpolation) and then we should be good to go!